custom/icds_reports/management/commands/sql_scripts/ap_data_pull_1.sql

```diff
@@ -0,0 +1,65 @@
+COPY(SELECT
+    awc.state_name,
+    awc.district_name,
+    awc.block_name,
+    awc.supervisor_name,
+    awc.awc_name,
+    awc.awc_site_code,
+    CASE WHEN (SUM(CASE WHEN chm.nutrition_status_weighed=1 AND chm.age_tranche::int<=36 AND month='%(month_1)s' THEN 1 ELSE 0 END)/SUM(CASE WHEN chm.wer_eligible=1 AND chm.age_tranche::int<=36 AND month='%(month_1)s' THEN 1 ELSE 0 END))::float>=0.6 THEN 'Y'
+        ELSE 'N'
+    END as weight_%(column_1)s,
+    SUM(CASE WHEN chm.num_rations_distributed>=21 AND month='%(month_1)s' THEN 1 ELSE 0 END) as thr_count_%(column_1)s,
+    SUM(CASE WHEN chm.thr_eligible=1 AND month='%(month_1)s' THEN 1 ELSE 0 END) as thr_eligible_%(column_1)s,
+
+    CASE WHEN (SUM(CASE WHEN chm.nutrition_status_weighed=1 AND chm.age_tranche::int<=36 AND month='%(month_2)s' THEN 1 ELSE 0 END)/SUM(CASE WHEN chm.wer_eligible=1 AND chm.age_tranche::int<=36 AND month='%(month_2)s' THEN 1 ELSE 0 END))::float>=0.6 THEN 'Y'
+        ELSE 'N'
+    END as weight_%(column_2)s,
+    SUM(CASE WHEN chm.num_rations_distributed>=21 AND month='%(month_3)s' THEN 1 ELSE 0 END) as thr_count_%(column_2)s,
+    SUM(CASE WHEN chm.thr_eligible=1 AND month='%(month_2)s' THEN 1 ELSE 0 END) as thr_eligible_%(column_2)s,
+
+    CASE WHEN (SUM(CASE WHEN chm.nutrition_status_weighed=1 AND chm.age_tranche::int<=36 AND month='%(month_3)s' THEN 1 ELSE 0 END)/SUM(CASE WHEN chm.wer_eligible=1 AND chm.age_tranche::int<=36 AND month='%(month_3)s' THEN 1 ELSE 0 END))::float>=0.6 THEN 'Y'
+        ELSE 'N'
+    END as weight_%(column_3)s,
+    SUM(CASE WHEN chm.num_rations_distributed>=21 AND month='%(month_3)s' THEN 1 ELSE 0 END) as thr_count_%(column_3)s,
+    SUM(CASE WHEN chm.thr_eligible=1 AND month='%(month_3)s' THEN 1 ELSE 0 END) as thr_eligible_%(column_3)s
+FROM
+    "awc_location" awc LEFT JOIN "child_health_monthly" chm ON (
+        chm.supervisor_id=awc.supervisor_id and
+        chm.awc_id = awc.doc_id
+    )
+    where chm.month in ('%(month_1)s','%(month_2)s','%(month_3)s')
+    and awc.state_id='f98e91aa003accb7b849a0f18ebd7039' and awc.aggregation_level=5
+GROUP by awc.state_name,
+    awc.district_name,
+    awc.block_name,
+    awc.supervisor_name,
+    awc.awc_name,
+    awc.awc_site_code) TO '/tmp/%(name)s/ap_data_pull_1.csv' DELIMITER ',' CSV HEADER
```

Comment:
will this give error if these folders are not already created?

Reply:
Not sure, let's try something basic on the server